python/src/FittingTest_doc.i.in

```diff
@@ -277,6 +283,13 @@ Normal(mu = 0, sigma = 1)"
 %feature("docstring") OT::FittingTest::BIC
 "Compute the Bayesian information criterion.
 
+
+If a :class:`~openturns.Distribution` is used, the likelihood is 
+evaluated on the sample. 
+If a :class:`~openturns.DistributionFactory` is used, the 
```

Comment:
```suggestion
If a :class:`~openturns.DistributionFactory` is used, its
```
I think "its" is is a little clearer than "the" since on the HTML rendering, readers will not see "the DistributionFactory.build method" but "the build method" (which one?)